scripts/build

```diff
@@ -373,13 +373,14 @@ fi
 
 pkg_call_exists post_make_$TARGET && pkg_call post_make_$TARGET
 
-# make install writes directly to sysroot which then needs fixing up - this is racy
-if listcontains "configure:target cmake-make:target autotools:target make:target" "${PKG_TOOLCHAIN}:${TARGET}"; then
-  acquire_exclusive_lock "${PKG_NAME}:${TARGET}" "build"
-  PKG_NEED_UNLOCK=yes
-else
-  PKG_NEED_UNLOCK=no
-fi
+# Hack around directly writing/modifying the content of a shared sysroot
+# by temporarily installing new files to a package specific sysroot
+PKG_ORIG_SYSROOT_PREFIX="${SYSROOT_PREFIX}"
+export SYSROOT_PREFIX="${BUILD}/.sysroot/${PKG_NAME}.${TARGET}"
+rm -fr "${SYSROOT_PREFIX}"
```

Comment:
This seems scary but I think it's safe

Reply:
Yes, hopefully I got everything in the right order. :)